libraries/robin_hood/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 
 # RobinHood library
-add_library(robin_hood STATIC
+add_library(robin_hood INTERFACE
```

Comment:
Header only libraries need to be added as interfaces or cmake will attempt to link to a non-existent built library